bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -438,7 +438,10 @@ async def _edit_nomination_reason(
             json={"actor": actor.id, "reason": reason}
         )
         await self.refresh_cache()  # Update cache
-        await ctx.send(":white_check_mark: Successfully updated nomination reason.")
+        await ctx.send(
+            f":white_check_mark: Successfully updated nomination reason"
```

Comment:
How about extending this behaviour of mentioning the nominee to cases where the nomination is inactive ([line 427](https://github.com/python-discord/bot/blob/762dca11064d2487fa0c7fc3be4284775e19c293/bot/exts/recruitment/talentpool/_cog.py#L427), might need a slight rewording in that case) and when the actor has not nominated the user ([line 431](<https://github.com/python-discord/bot/blob/762dca11064d2487fa0c7fc3be4284775e19c293/bot/exts/recruitment/talentpool/_cog.py#L431>))?